tests/testVisitInfo.py

```diff
@@ -195,10 +195,10 @@ def testSetVisitInfoMetadata(self):
 
     def testSetVisitInfoMetadataMissingValues(self):
         """If a value is unknown then it should not be written to the metadata"""
-        visitInfo = afwImage.makeVisitInfo()  # only rot type is known
+        visitInfo = afwImage.VisitInfo()  # only rot type is known
```

Comment:
We do not get the same warnings with SWIG now. With SWIG we have to use makeVisitInfo, so it doesn't print a deprecation warning. The change in pybind11 is the replace `makeVisitInfo` with `VisitInfo` (because pybind11 allows a wrapping with keyword arguments that have sensible defaults) and add a deprecation warning to `makeVisitInfo`. That is why I'm so keen to keep this new code. It is directly relevant to the work i did on this ticket.